src/Services/ExpressCheckout.php

```diff
@@ -138,6 +138,27 @@ public function doExpressCheckoutPayment($data, $token, $payerid)
 
         return $response;
     }
+    
+    /**
+     * Function to perform DoCapture PayPal API operation.
+     *
+     * @param  string $authorization_id Transaction ID
+     * @param  float  $amount           Amount to capture
+     * @param  string $complete         Indicates whether or not this is the last capture.
+     * @param  array  $data             Optional request fields
+     * @return array
+     */
+    public function doCapture($authorization_id, $amount, $complete = 'Complete', $data = [])
```

Comment:
What if you would like to add a `NOTE`, for example, or any other optional parameter? https://developer.paypal.com/docs/classic/api/merchant/DoCapture_API_Operation_NVP/